Pod/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -528,6 +532,17 @@ - (void)applicationWillTerminate
     }];
 }
 
+-(void)applicationDidFinishLaunching
```

Comment:
I think it's more appropriate for migration to happen at `SEGSegmentIntegration` initialization time rather than waiting until `applicationDidFinishLaunching`. The important bit here is to ensure that old events are migrated prior to the rest of `SEGSegmentIntegration` functionality happening.